packages/account-v2/src/constants/manualFormConstants.tsx

```diff
@@ -9,18 +9,18 @@ import ProofOfAgeIcon from '../assets/manual-upload/proof-of-age.svg';
 export type TManualDocumentTypes = typeof MANUAL_DOCUMENT_TYPES[keyof typeof MANUAL_DOCUMENT_TYPES];
 
 export const MANUAL_DOCUMENT_TYPES = Object.freeze({
-    DRIVING_LICENCE: 'driving_licence',
-    NATIONAL_IDENTITY_CARD: 'national_identity_card',
-    NIMC_SLIP: 'nimc_slip',
-    PASSPORT: 'passport',
+    DRIVING_LICENCE: 'DRIVING_LICENCE',
+    NATIONAL_IDENTITY_CARD: 'NATIONAL_IDENTITY_CARD',
+    NIMC_SLIP: 'NIMC_SLIP',
+    PASSPORT: 'PASSPORT',
```

Comment:
Can we follow AirBnb style for exported objects

export const MAPPING = {
  key: 'value',
};

cc: @arshad-rao-deriv